app/src/androidTest/java/edu/nextstep/camp/calculator/MainActivityTest.kt

```diff
@@ -13,81 +13,118 @@ class MainActivityTest {
     @get:Rule
     var activityScenarioRule = ActivityScenarioRule(MainActivity::class.java)
 
-    //사용자가 피연산자 0 ~ 9 버튼을 누르면 화면에 해당 숫자가 화면에 보여야 한다.
+    @Test
+    fun 버튼_5과_더하기와_1를_누르면_화면에_그대로_수식이_보여야한다() {
+        //when
+        onView(ViewMatchers.withId(R.id.button5)).perform(click())
+        onView(ViewMatchers.withId(R.id.buttonPlus)).perform(click())
+        onView(ViewMatchers.withId(R.id.button1)).perform(click())
+        //then
+        onView(ViewMatchers.withId(R.id.tv_result)).check(ViewAssertions.matches(withText("5 + 1")))
+    }
```

Comment:
ViewMatchers.withId, ViewAssertions.matches 도 다른 함수와 동일하게 static import 하시면 조금더 깔끔한 코드가 될것 같습니다. ✌

```suggestion
    fun 버튼_5과_더하기와_1를_누르면_화면에_그대로_수식이_보여야한다() {
        //when
        onView(withId(R.id.button5)).perform(click())
        onView(withId(R.id.buttonPlus)).perform(click())
        onView(withId(R.id.button1)).perform(click())
        //then
        onView(withId(R.id.tv_result)).check(matches(withText("5 + 1")))
    }
```